auth-web/src/routes/router.ts

```diff
@@ -187,6 +188,11 @@ export function getRoutes (): RouteConfig[] {
           meta: {
             isPremiumOnly: true
           }
+        },
+        {
+          path: 'product-package',
```

Comment:
I think we are using product-settings as the route in settings endpoint, can you change route to product-settings